src/invidious/config.cr

```diff
@@ -52,6 +52,7 @@ struct ConfigPreferences
   property vr_mode : Bool = true
   property show_nick : Bool = true
   property save_player_pos : Bool = false
+  property exclude_members_only_videos : Bool = false
```

Comment:
Should we exclude the channel only videos by default?

Reply:
Honnestly, I don't see a reason to show member-only videos, as Invidious users can't watch them anyway. We always create a dedicated channel page/place for the various "special" playlists (`UUMO` and co).